### DISCUSSION TITLE
[FEATURE-REQUEST] Dejar un poco más centralizado los botones "Enviar y Cancelar" respecto al formulario, cuando éste se levanta

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#256] [3. Ideas & Feature Requests] [answered]
Sería bueno que ese campo sea un poco más cerca al formulario ya que se ve un poco descolocado y abarca mayor el campo visual de un usuario

**Actualmente**

![image](https://user-images.githubusercontent.com/61419471/121602378-378c5600-ca15-11eb-893a-b59f87284b36.png)

**Idealmente alternativa 1**

![image](https://user-images.githubusercontent.com/61419471/121602488-5c80c900-ca15-11eb-870b-008c6ec974ad.png)

**Idealmente alternativa 2**

![image](https://user-images.githubusercontent.com/61419471/121602541-70c4c600-ca15-11eb-8922-4c16b0c4270e.png)

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@acedward que opinas de esto?
